poutyne/framework/callbacks/lr_scheduler.py

```diff
@@ -12,10 +12,9 @@
 from .callbacks import Callback
 
 
-class _PyTorchLRSchedulerWrapper(Callback):
-    def __init__(self, torch_lr_scheduler, *args, **kwargs):
+class LRScheduler(Callback):
```

Comment:
The template was mainly introduced as a reason to differentiate between LR scheduling callbacks and normal ones, since the former one requires a call to load_state to be made when retraining a network.

The actual class does not actually support a typical heritage pattern where you would need to force implementation of methods for example. Its most valuable usages are 1) being able to refer to it for the lr_schedulers argument to the train method in Experiment and 2) code reutilization.

I have added documentation on the class in da0ee773d9a7a0d4afeab3f843623fbe9ae133d9.